cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -74,7 +74,7 @@ else ()
   set(SNAPPY_INCLUDE_DIR "${SNAPPY_HOME}/include")
   set(SNAPPY_STATIC_LIB "${SNAPPY_HOME}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}snappy${CMAKE_STATIC_LIBRARY_SUFFIX}")
   set(SNAPPY_CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${SNAPPY_HOME}
-                        -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_LIBDIR=lib)
+                        -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_LIBDIR=lib -DSNAPPY_BUILD_TESTS=OFF)
```

Comment:
This looks like a regression. Why do we need this?

Reply:
> This looks like a regression. Why do we need this?

Snappy compiles the unittest will have a link error not founding many symbols, so not to compile the ut.

Reply:
Do you know why only `clang-13` fail with the link error?
> have a link error

Reply:
> 

It is also failure in gcc-9. The whole link message is i
[snappy_ep-build-err.log](https://github.com/apache/orc/files/7878515/snappy_ep-build-err.log)
n file. I think it is the linker that can not find gtest lib which I did not install. So It it better not compile the snappy unit test because we just use it not develop it.

Reply:
> Do you know why only `clang-13` fail with the link error?
> 
> > have a link error

It turns out that it's my env error with incomplete gtest lib. Thanks for your review.